src/transformers/models/conditional_detr/image_processing_conditional_detr.py

```diff
@@ -1434,8 +1433,8 @@ def preprocess(
                 return_pixel_mask=True,
                 data_format=data_format,
                 input_data_format=input_data_format,
-                return_tensors=return_tensors,
                 update_bboxes=do_convert_annotations,
+                return_tensors=return_tensors,
```

Comment:
This is just reordering as `return_tensors` is normally returned last and it makes it easier to compare models when all arguments are in the same order